apps/authentication/mixins.py

```diff
@@ -577,7 +593,8 @@ def clear_auth_mark(self):
         keys = [
             'auth_password', 'user_id', 'auth_confirm_required',
             'auth_notice_required', 'auth_ticket_id', 'auth_acl_id',
-            'user_session_id', 'user_log_id', 'can_send_notifications'
+            'user_session_id', 'user_log_id', 'can_send_notifications',
+            'next',
         ]
         for k in keys:
             self.request.session.pop(k, '')
```

Comment:
I'm sorry but it appears that you haven't provided actual code for analysis. Please do share the relevant piece of code so I can assist with checking its accuracy and any potential optimizations/issue detection.